src/emmy/quaternion.cljc

```diff
@@ -404,17 +404,6 @@
        (-invoke [this a b c d e f g h i j k l m n o p q r s t rest]
                 (evaluate this (into [a b c d e f g h i j k l m n o p q r s t] rest)))]))
 
-#_{:clj-kondo/ignore [:redefined-var]}
-(do (ns-unmap 'emmy.quaternion '->Quaternion)
```

Comment:
this was causing a build error in emmy-viewers. I thought I was being clever by allowing metadata to be smuggled in. But using `make` everywhere that we want to avoid passing in the metadata argument is the proper approach.